GlobalSnapshots/src/main/java/Message.java

```diff
@@ -1,11 +1,9 @@
 public class Message {
     public int id;
     public String command;
-    public int payload;
     
-    public Message(int id, String command, int payload){
+    public Message(int id, String command){
         this.id = id;
         this.command = command;
-        this.payload = payload;
```

Comment:
when compared to my branch: is there a reason you took out the marker from a message?
it creates a little more if statements and sorta deviates from overloading a receive function.  might make it a little harder to do a one-to-one with Gargs algorithm in our presentation.